object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+function omit(source, keys) {
+  const result = {};
+  for (const prop in source) {
+    for (let i = 0; i < keys.length; i++) {
```

Comment:
You can remove this inner `for` loop since `includes` loops through the keys array for you.